tests/unit/lifecycle/LifecycleTask.spec.js

```diff
@@ -2052,4 +2054,80 @@ describe('lifecycle task helper methods', () => {
                 });
         });
     });
+
+    describe('_retryEntry', () => {
+        it('should not retry on success', done => {
+            const lct = new LifecycleTask(lp);
+            const action = sinon.stub();
+            action.callsArg(0);
+
+            lct._retryEntry({
+                logFields: {},
+                log: fakeLogger,
+                actionFunc: action,
+            }, err => {
+                assert.ifError(err);
+                assert(action.calledOnce);
+                done();
+            });
+        });
+
+        it('should not retry if error is not retriable', done => {
+            const lct = new LifecycleTask(lp);
+            const action = sinon.stub();
+            action.callsArgWith(0, { code: 'NotRetriable', message: 'test' });
```

Comment:
If it's not too involved to fix the spelling ;)

```suggestion
            action.callsArgWith(0, { code: 'NotRetryable', message: 'test' });
```